olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/PartitionsRequest.h

```diff
@@ -50,28 +52,36 @@ class DATASERVICE_READ_API PartitionsRequest final {
   }
 
   /**
-   * @brief Get the catalog metadata version requested for the partitions.
-   * @return the billing tag, or boost::none if not set.
+   * @brief Gets the catalog metadata version of the requested partitions.
+   *
+   * @return The catalog metadata version.
    */
   inline const boost::optional<std::int64_t>& GetVersion() const {
     return catalog_metadata_version_;
   }
 
   /**
-   * @brief BillingTag is an optional free-form tag which is used for
-   * grouping billing records together. If supplied, it must be between 4 - 16
-   * characters, contain only alpha/numeric ASCII characters  [A-Za-z0-9].
-   * @return the billing tag, or boost::none if not set.
+   * @brief Gets a billing tag to group billing records together.
+   *
+   * The billing tag is an optional free-form tag that is used for grouping
+   * billing records together. If supplied, it must be 4&ndash;16 characters
```

Comment:
```suggestion
   * billing records together. If supplied, it must be 4-16 characters
```
Would a real dash, i.e. `-`, be cleaner? The encoding is nice for HTML but it's harsh on humans:)

Reply:
Agree.
Added the en dash sign (–) instead of `&ndash;`.